gamemodes/terrortown/gamemode/shared/lang/de.lua

```diff
@@ -1296,3 +1296,12 @@ Beachte, dass diese Übersetzungen Communitybasiert sind. Hilf mit, wenn Du Fehl
 
 -- 2021-04-20
 L.none = "Keine Rolle"
+
+-- 2021-04-24
+L.karma_teamkill_tooltip = "Karma erhalten durch Töten eines Team-Mitglieds."
+L.karma_teamhurt_tooltip = "Karma erhalten durch Verursachen von Schaden an einem Team-Mitglied."
+L.karma_enemykill_tooltip = "Karma erhalten durch Töten eines Feindes."
+L.karma_enemyhurt_tooltip = "Karma erhalten durch Verursachen von Schaden an einem Feind."
+L.karma_cleanround_tooltip = "Karma erhalten durch eine einwandfreie Runde."
+L.karma_roundheal_tooltip = "Karma erhalten am Rundenende."
+L.karma_unknown_tooltip = "Unbekanntes Ereignis."
```

Comment:
Same as in the english lang file: possibly missing karma context

```suggestion
L.karma_unknown_tooltip = "Unbekanntes Karma Ereignis."
```